docs/reference/openapi-rules.md

```diff
@@ -1,3 +1,546 @@
-## OpenAPI Rules
+# OpenAPI Rules
 
-Coming soon...
\ No newline at end of file
+Spectral has three rulesets built in: 
+
+- [oas](#oas)
+- [oas2](#oas2)
+- [oas3](#oas3)
+
+_OAS is shorthand for OpenAPI Specification._
+
+In your ruleset file you can add `extends: "spectral:oas"` and you'll get everything from `spectral:oas2` and `spectral:oas3`, meaning you can lint either type of document and only the relevant rules will apply thanks to [formats](../getting-started/rulesets.md#formats).
+
+Let's look at the rules in these rulesets.
+
+## oas
+
+### operation-2xx-response
+
+Operation must have at least one `2xx` response. Any API operation (endpoint) can fail but presumably it is also meant to do something constructive at some point. If you forget to write out a success case for this API, then this rule will let you know.
+
+**Recommended:** Yes
+
+**Bad Example**
+
+```yaml
+paths:
+  /path:
+    get:
+      responses:
+        418:
+          description: teapot
+```
+
+### operation-operationId-unique
+
+Every operation must have a unique `operationId`. 
+
+Why? A lot of documentation systems use this as an identifier, some SDK generators convert them to a method name, all sorts of things like that. Giving each one a unique name avoids all sorts of problems.
```

Comment:
`all sorts of things like that` and `all sorts of problems`, hm.. does this add value to the doc?

Reply:
Thanks, trimmed the last sentence. Trying to explain a lot of shit can go wrong whilst not actually remembering a third example 😆